dependencies/precise/hammer_cli/control

```diff
@@ -21,7 +21,7 @@ Depends: ${shlibs:Depends}, ${misc:Depends}, ruby | ruby-interpreter,
          ruby-fastercsv,
          ruby-mime-types (< 2.0.0),
          ruby-fast-gettext (>= 0.8.0),
-         liblocale-ruby1.8 (<= 2.0.9) | liblocale-ruby1.9.1 (<= 2.0.9),
+         liblocale-ruby1.8 (>= 2.0.6) | liblocale-ruby1.9.1 (>= 2.0.6),
```

Comment:
12.04 seems to have 2.0.5: http://packages.ubuntu.com/search?keywords=liblocale-ruby